### ISSUE TITLE
Support Ace Attorney engine

### ISSUE THREAD

**Issue by henke37** [monyarm/NovelVM#2]
## Engine

Ace Attorney / Gyakuten Saiban proprietary engine(s)

## Description

Support the proprietary engine(s) used by the Ace Attorney series. The original releases require reverse engineering from scratch. The PC/PS4/Switch trilogy rerelease uses Unity and may need to be treated as a separate implementation target.

## Games

- [Gyakuten Saiban / Ace Attorney series](https://vndb.org/v711)

## Repository URLs

- None known

## Reverse-engineering documentation

- None known

**Comment by henke37** [comment]
I think this was closed by accident.

**Comment by monyarm** [comment]
Yup, AI agent closed this while squashing commiits, cause a scummvm commit had the same number and was referenced
